notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -13,6 +13,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        (
</pre><p>Expression is long and nested which makes it hard to read. The <code>.groupby()\n.mean()</code> pattern seems fine (it's like a pie, each step follows sequentially) but that <em>plus</em> multiplication and then plotting on the outside is a bit much. Consider introducing temporary variable to hold the <code>adv_win_rate</code> then rescale by <code>100</code> and plot? There's also some code duplication here -- it's just the query that varies between lines 20-29 and 30-36 -- so you could probably factor that out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>